database/queries.ts

```diff
@@ -134,6 +134,43 @@ function calculateStreak(entryTimes: number[], currentTime :number, waitTime :nu
   return streak;
 }
 
+export const readInventory = async () => {
+  const db = await dbPromise;
+  const rows : {InventoryOfItems:string}[]= await db.getAllAsync('SELECT InventoryOfItems FROM App');
+  console.log(rows);
+
+  const inventoryString = rows[0].InventoryOfItems;
+  const inventory = JSON.parse(inventoryString);
+
+  return(inventory);
+}
+
+export const updateInventory = async (inventory: ItemAndCount[]) => {
+  const db = await dbPromise;
+  const result = await db.runAsync('UPDATE App SET InventoryOfItems = ?', JSON.stringify(inventory));
+  console.log(`Updated ${result.changes} row(s)`);
+}
+
+export const getCurrencyAmount = async () => {
+  const db = await dbPromise;
+  const rows: {CurrencyAmount: number}[] = await db.getAllAsync('SELECT CurrencyAmount FROM App');
+  return rows[0].CurrencyAmount;
+}
+export const reduceCurrencyAmount = async (amount: number) => {
+  const db = await dbPromise;
+  const rows : {CurrencyAmount: number}[]= await db.getAllAsync('SELECT CurrencyAmount FROM App');
+  const newAmount = rows[0].CurrencyAmount - amount;
+  const result = await db.runAsync('UPDATE App SET CurrencyAmount = ?', newAmount);
+  console.log(`Updated ${result.changes} row(s)`);
+}
+
+export const addCurrencyAmount = async (amount: number) => {
+  const db = await dbPromise;
+  const rows : {CurrencyAmount: number}[]= await db.getAllAsync('SELECT CurrencyAmount FROM App');
+  const newAmount = rows[0].CurrencyAmount + amount;
+  const result = await db.runAsync('UPDATE App SET CurrencyAmount = ?', newAmount);
+  console.log(`Updated ${result.changes} row(s), current currency amount: ${newAmount}`);
+}
```

Comment:
Are we wanting an offline store as well? If we *don't*, then we also don't need to worry about this client side.